app/controllers/EightBallController.scala

```diff
@@ -0,0 +1,30 @@
+package controllers
+
+import javax.inject._
+
+import play.api.mvc._
+
+import models.EightBallReply
+
+@Singleton
+class EightBallController @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
+
+  def randomReply = Action {
+    val rndString = EightBallReply.getRndBall.content
+    Ok(views.txt.text(rndString))
+  }
+
+  def getMessages = Action {
```

Comment:
Can you change this to `messages`, please? The way it is now is more Java than Scala.